src/main/java/io/github/externschool/planner/service/ScheduleEventTypeServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package io.github.externschool.planner.service;
+
+import io.github.externschool.planner.entity.schedule.ScheduleEventType;
+import io.github.externschool.planner.repository.schedule.ScheduleEventTypeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+/**
+ * @author Danil Kuznetsov (kuznetsov.danil.v@gmail.com)
+ */
+@Service
+@Transactional
```

Comment:
i suppose this  `@Transactional` annotation is unnecessary here, when Spring Data is used:
```By default, CRUD methods on repository instances are transactional. For read operations, the transaction configuration readOnly flag is set to true. All others are configured with a plain @Transactional so that default transaction configuration applies.``` https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#transactions
There in docs they mention 3 issues when `@Transactional` is required:
1. To tweak transaction configuration for one of the methods declared in a repository (e.g. `@Transactional(timeout = 10)`)
2. ``Another way to alter transactional behaviour is to use a facade or service implementation that (typically) covers more than one repository. Its purpose is to define transactional boundaries for non-CRUD operations. ``
3. ``To let your query methods be transactional``

Reply:
I said about this case yesterday

Reply:
thanks for your comprehensive explanation